opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicMetricExporter.java

```diff
@@ -195,10 +195,10 @@ private Attributes buildCommonAttributes(MetricData metric) {
     }
 
     /**
-     * Set a URI to override the default ingest endpoint.
+     * Set a URI to override the default metric ingest endpoint. If this is not provided, the
+     * telemetry SDK will choose the US production endpoint as a default.
      *
-     * @param uriOverride The scheme, host, and port that should be used for the Metric API
-     *     endpoint. The path component of this parameter is unused.
+     * @param uriOverride - The fully qualified URI endpoint for the metric ingest api.
```

Comment:
so...what do we do with users who are using the other way? I think we should at least create a new method and deprecate this one, rather than break everyone who is using the override already.

Reply:
Hmmm.  In the pre-1.0 release world and given how young this is, I thought it would be acceptable.  I kinda hate to leave that cruft around.  When/how would we get rid of it?  After the sdk removes it?

Reply:
I'm with Rich Hickey and Steve Yegge..never remove it. Just document that it's a sub-optimal solution and keep it there.

Reply:
To paraphrase: "don't make your customer's life hard just because you made a bad decision and regret it. It's not their fault"